lib/measures/tbd/measure.xml

```diff
@@ -254,5 +247,21 @@
       <usage_type>license</usage_type>
       <checksum>4AB09951</checksum>
     </file>
+    <file>
+      <filename>README.md</filename>
+      <filetype>md</filetype>
+      <usage_type>readme</usage_type>
+      <checksum>54C6A3C1</checksum>
+    </file>
+    <file>
+      <filename>psi.rb</filename>
+      <filetype>rb</filetype>
+      <usage_type>resource</usage_type>
+<<<<<<< HEAD
```

Comment:
@rd2 more merge conflicts

Reply:
All tests run fine. Should be merging the pull request after some final checks and tests. Thanks again.